tests/datasource/fluent/test_spark_filesystem_datasource.py

```diff
@@ -83,6 +83,25 @@ def test_add_csv_asset_to_datasource(
     assert m1 is not None
 
 
+# TODO: Parametrize with all asset types
+@pytest.mark.unit
+def test_add_parquet_asset_to_datasource(
+    spark_filesystem_datasource: SparkFilesystemDatasource,
+):
+    asset = spark_filesystem_datasource.add_parquet_asset(
+        name="parquet_asset",
+        datetime_rebase_mode="EXCEPTION",
```

Comment:
@anthonyburdi To me an `Enum` would appear better, but I am not sure if this would add more work to users.

Reply:
That's a good point. I think since this is a pass through to spark and we have pydantic runtime validation for the acceptable values that a string may be good enough here. But I would love to hear team thoughts e.g. @Kilo59